molgenis-settings/src/main/java/org/molgenis/settings/entity/EntitySettingsPackage.java

```diff
@@ -0,0 +1,36 @@
+package org.molgenis.settings.entity;
+
+import static java.util.Objects.requireNonNull;
+import static org.molgenis.data.system.model.RootSystemPackage.PACKAGE_SYSTEM;
+
+import org.molgenis.data.meta.SystemPackage;
+import org.molgenis.data.meta.model.PackageMetadata;
+import org.molgenis.data.system.model.RootSystemPackage;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class EntitySettingsPackage extends SystemPackage {
```

Comment:
**suggestion:** Can we use another name for this package?

_Entity_ has become a word with many meanings and I try to keep as far from it as I can to avoid further confusion. In this case the package will store settings that are not 'global' (like the other, existing settings), but settings that can be configured on an EntityType level, or in the future maybe on a Group or User level. I'll be honest: I don't have an alternative, but if anything pops up I'll let you know. Anyways, _entity_ is a bit too vague here in my opinion.

Reply:
renamed to 'tableSettingsPackage'